src/main/java/org/gridsuite/study/server/service/ConsumerService.java

```diff
@@ -437,7 +439,7 @@ public Consumer<Message<String>> consumeShortCircuitAnalysisFailed() {
                     // send notification for failed computation
                     UUID studyUuid = networkModificationTreeService.getStudyUuidForNodeId(receiverObj.getNodeUuid());
 
-                    notificationService.emitStudyChanged(studyUuid, receiverObj.getNodeUuid(), NotificationService.UPDATE_TYPE_SHORT_CIRCUIT_FAILED);
+                    notificationService.emitStudyError(studyUuid, receiverObj.getNodeUuid(), NotificationService.UPDATE_TYPE_SHORT_CIRCUIT_FAILED, errorMessage, userId);
```

Comment:
UPDATE_TYPE_SHORT_CIRCUIT_FAILED is lost for others users working on this study. 
Better send 2 notifications ? One to the study, on the user that ran the analysis ?

Reply:
Yes as discussed I forgot this case, I'm doing it